kolla-ansible/ansible/roles/triliovault/templates/fuse.conf.j2

```diff
@@ -0,0 +1,6 @@
+# /etc/fuse.conf - Configuration file for Filesystem in Userspace (FUSE)
+# Set the maximum number of FUSE mounts allowed to non-root users.
+# The default is 1000.
+mount_max = 2000
+# Allow non-root users to specify the allow_other or allow_root mount options.
+user_allow_other
```

Comment:
We need to add a empty line at the end of this file. Otherwise it throws error.

Reply:
Okay, will add it.